fe/estimator_abfe.py

```diff
@@ -373,6 +373,7 @@ def _deltaG_from_results(model, results, sys_params) -> Tuple[Tuple[float, List]
             beta=model.beta,
             lhs_xs=results[-2].xs,
             rhs_xs=results[-1].xs,
+            seed=2021,
```

Comment:
This commit may not make it, but would make the RABFE protocol deterministic all the way through. Note that the inputs still change as a function of the seed of the simulation.